03_ScienceExamples/SpectralEnergyDistributions/SpectralEnergyDistributionsInSplusDr1.ipynb

```diff
@@ -350,7 +350,7 @@
     "\n",
     "The many S-PLUS photometric bands can be used to produce color-color diagrams that are useful for measuring properties such as the metallicity or surface gravity of a star.\n",
     "\n",
-    "(g-F515) vs. (g-r) is sensitive to surface gravity and can be used to separate dwarf stars from giant stars.  The main locus of points is the dwarfs will the red giant live in the upper right.  [See Majewski et al. (2000)](http://adsabs.harvard.edu/abs/2000AJ....120.2550M) for more details on this method."
+    "(g-F515) vs. (g-r) is sensitive to surface gravity and can be used to separate dwarf stars from giant stars.  The main focus of points is the dwarfs, while the red giant live in the upper right.  [See Majewski et al. (2000)](http://adsabs.harvard.edu/abs/2000AJ....120.2550M) for more details on this method."
```

Comment:
'locus' was correct, please change back. The sentence had typos though, and I think it should be "The main locus of points is the dwarfs, while the red giants live in the upper right."